daemon/cluster/cluster.go

```diff
@@ -134,6 +141,14 @@ func New(config Config) (*Cluster, error) {
 	if config.RuntimeRoot == "" {
 		config.RuntimeRoot = root
 	}
+	if config.RaftHeartbeatTick == 0 {
+		config.RaftHeartbeatTick = 1
+	}
+	if config.RaftElectionTick == 0 {
+		// 10X heartbeat tick is the recommended ratio according to etcd docs.
+		config.RaftElectionTick = 10 * config.RaftHeartbeatTick
```

Comment:
What happens if I configure `RaftElectionTick < RaftHeartbeatTick`? Should `RaftElectionTick` always be multiple times `RaftHeartbeatTick`? If so; should we add validation here, or have a `ratio` instead as configuration?

Reply:
This here doesn't enforce that `RaftElectionTick` should always be > than `RaftHearbeatTick` and should be at least 10 (which is the default value when nothing is set).  

I can add a check to return an error when `RaftElectionTick` is strictly greater than `RaftHeartbeatTick` and prevent the daemon from starting up.  Is this acceptable?